packages/editor/src/init.tsx

```diff
@@ -100,6 +113,7 @@ export function initSunmaoUIEditor(props: SunmaoUIEditorProps = {}) {
     widgetManager,
     eventBus,
     editorStore,
+    doOperations,
```

Comment:
Can we move this function to `appModelManager`? Because this function is quiet different with other services.

Reply:
Done.